### PR TITLE
Allow duplicate properties (the latest property wins)

### PR DESCRIPTION
**Pull request by vkartaviy**

### CODE REVIEW

File: rrule/src/tests/regression.rs [hunk 1]
```diff
@@ -260,6 +260,24 @@ fn issue_america_edmonton_ambiguous_date_on_dst_switch_off() {
     common::check_occurrences(&dates, &["2020-11-01T01:00:00-06:00"]);
 }
 
+#[test]
+fn duplicate_property_for_until_should_use_latest_one() {
```
Comment (ivopintodasilva) [inline rrule/src/tests/regression.rs:264]: should we use the latest one or the first one? 🤔 

it's such an edge case that we can probably just do what's easier for us...

**Review by ivopintodasilva — APPROVED** [review]
